awx/ui/client/test/e2e/commands/inject.js

```diff
@@ -0,0 +1,22 @@
+exports.command = function(deps, script, callback) {
+    let self = this;
```

Comment:
There's a mix of `self` and `this` -- are there different object contexts at play here?

Reply:
This used to be the case but no longer. Removed.